src/Expander/TC.hs

```diff
@@ -253,38 +253,38 @@ unifyWithBlame blame depth t1 t2 = do
             (Ty (TyF (TMetaVar ptr2) (take (argCount2 - argCount) args2)))
 
     -- Flex-rigid
-    unifyTyFs expected@(TyF (TMetaVar ptr1) args1) received@(TyF ctor2 args2)
+    unifyTyFs shouldBe@(TyF (TMetaVar ptr1) args1) received@(TyF ctor2 args2)
       | null args1 =
         linkVar ptr1 t2
       | length args1 <= length args2 = do
           traverse_ (uncurry $ unifyWithBlame blame (depth + 1)) $
             zip args1 (drop (length args2 - length args1) args2)
           let args2' = take (length args2 - length args1) args2
           unifyWithBlame blame (depth + 1) (Ty $ TyF (TMetaVar ptr1) []) (Ty $ TyF ctor2 args2')
-      | otherwise = liftIO (putStrLn "hey") >> mismatch expected received
+      | otherwise = liftIO (putStrLn "hey") >> mismatch shouldBe received
```

Comment:
Huh, what's this `hey` doing there? Looks like a stray debugging line from an earlier PR, let's remove it while we're here

Reply:
So responsive!

Reply:
Also, this `mismatch` function should probably ask for `Actual` and `Expected` arguments as well